tasks/collect_files_to_commit.js

```diff
@@ -0,0 +1,15 @@
+import filesToCommit from './utils/files_to_commit';
+import { resolve } from 'path';
+
+const root = resolve(__dirname, '..');
+
+export default function registerCollectFilesToCommit(grunt) {
+  grunt.registerTask('collectFilesToCommit', function () {
+    filesToCommit(root)
```

Comment:
Is that a thing?  I just pulled `root` directly from the old `collect_staged_filenames` task.